### DISCUSSION TITLE
Breaking changes in 1.1.0

### DISCUSSION THREAD

**Discussion by jonathanfox5** [SYSTRAN/faster-whisper#1180] [Q&A] [answered]
I was wondering if there was a list of breaking changes between 1.1.0 and 1.0.3?

I'm trying to get whisperX working with the latest update and it seems that the only change that breaks whisperX is the addition of `transcriber.TranscriptionOptions.multilingual`. However, I'd be interested if anyone knows of any more that I might have missed?

**Comment by MahmoudAshraf97** [comment, marked as answer]
There are no breaking changes between 1.1.0 and 1.0.3
WhisperX is not maintained anymore and the latest version known to work out of the box is 1.0.0

**Reply by jonathanfox5** [reply]
> There are no breaking changes between 1.1.0 and 1.0.3 WhisperX is not maintained anymore and the latest version known to work out of the box is 1.0.0

Thanks, I suppose it's a whisperX implementation issue rather than a faster-whisper issue (I previously had to patch whisperx to get it to work with 1.0.3).

My reason for using whisperx is because of the accuracy of the wav2vec2 alignment. Is this something that is on the roadmap for faster-whisper or is there another project that you can recommend?

Thanks for all of your work on this project :)

**Reply by MahmoudAshraf97** [reply]
For forced alignment, check [This](https://github.com/MahmoudAshraf97/ctc-forced-aligner) which is a lightweight package that offers wav2vec2 alignment for any model and also supports multilingual alignments without needing a model for each language.

**Reply by jonathanfox5** [reply]
Awesome, thank you so much!
